modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -823,6 +823,87 @@ definition {
 		ObjectPortlet.viewEntry(entry = "Entry B");
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to edit the relationships of the Custom Object entry (One-to-Many)"
+	@priority = "4"
+	test CanEditOneToManyRelationshipOfCustomObjectEntries {
+		task ("Add a Custom Object, a field, create a 1toM relationship with the User object, and add an entry (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 293",
+				objectName = "CustomObject293",
+				pluralLabelName = "Custom Objects 293");
```

Comment:
To Custom Object, add the number ticket correct. Would be: 
`Custom Object 156677`
`CustomObject156677`
`Custom Objects 156677`

Reply:
After change to all Custom Objects